app/views/schools/dashboard/_school_cohort_details.html.erb

```diff
@@ -3,7 +3,9 @@
 <% if school_cohort.school_chose_cip? && school_cohort.default_induction_programme&.core_induction_programme.blank? %>
   <h2 class="govuk-heading-m">Choose your training materials</h2>
   <p class="govuk-body"><%= govuk_link_to "Tell us which materials you’ll use", info_schools_core_programme_materials_path(cohort_id: school_cohort.cohort.start_year) %></p>
-  <p class="govuk-body">Read our guidance to <%= govuk_link_to 'compare materials (opens in new tab)', page_path(page: "core-materials-info"), target: "_blank" %>.</p>
+  <p class="govuk-body">Read our guidance
+    to <%= govuk_link_to 'compare materials (opens in new tab)', "https://support-for-early-career-teachers.education.gov.uk", target: "_blank" %>
+    .</p>
```

Comment:
I think the full stop has to remain on the previous line to avoid a space appearing?

```suggestion
    to <%= govuk_link_to 'compare materials (opens in new tab)', "https://support-for-early-career-teachers.education.gov.uk", target: "_blank" %>.</p>
```